tokio/src/lib.rs

```diff
@@ -10,16 +10,11 @@
     unreachable_pub
 )]
 #![deny(unused_must_use)]
-#![cfg_attr(docsrs, deny(rustdoc::broken_intra_doc_links))]
 #![doc(test(
     no_crate_inject,
     attr(deny(warnings, rust_2018_idioms), allow(dead_code, unused_variables))
 ))]
 #![cfg_attr(docsrs, feature(doc_cfg))]
-#![cfg_attr(docsrs, feature(doc_cfg_hide))]
-#![cfg_attr(docsrs, doc(cfg_hide(docsrs)))]
-#![cfg_attr(docsrs, doc(cfg_hide(loom)))]
-#![cfg_attr(docsrs, doc(cfg_hide(not(loom))))]
```

Comment:
Have you verified that the documentation still looks alright after removing these?

Reply:
Yes

**nightly-2021-10-25, on taiki-e/doc_cfg branch**:

```
cd tokio
RUSTDOCFLAGS='--cfg docsrs' cargo +nightly-2021-10-25 doc --all-features --open
```

<img width="438" alt="10-25" src="https://user-images.githubusercontent.com/43724913/143000745-c38d5617-c6ca-4b21-9d1e-ad29379d8ed5.png">


**nightly-2021-11-23, on taiki-e/doc_cfg branch**:

```
cd tokio
RUSTDOCFLAGS='--cfg docsrs' cargo +nightly-2021-11-23 doc --all-features --open
```

<img width="438" alt="11-23" src="https://user-images.githubusercontent.com/43724913/143000765-538e5ed0-8548-4ba2-bc48-147df290adce.png">